notes/minimum_contracts.html

```diff
@@ -188,12 +188,13 @@ <h4><a name="analysis.going_in.on_off">Turning on and off all runtime checks</a>
     <p>We believe it is essential that we meet two somewhat contradictory requirements:</p>
     
     <ol>
-        <li>A guarantee that program never continues after a contract predicate is violated</li>
-        <li>A guarantee that runtime checking may be elided</li>
+        <li>A guarantee that some action is taken whenever a contract is violated</li>
+        <li>A guarantee that there is no runtime overhead for encoded contracts</li>
     </ol>
 
     <p>
-    The former is a correctness requirement, and the latter is performance requirement (after
```

Comment:
I still prefer the previous wording here.

Reply:
I rolled back this line.  We could go in circles nitpicking the details of this, but I think shorter is better at this point.

Reply:
Are we good with the outcome?